### PR TITLE
GUI-2691: Populate availability zone choices when zone is a CF template parameter

### PR DESCRIPTION
**Pull request by kamalgill**
Fixes https://eucalyptus.atlassian.net/browse/GUI-2691 (for 4.3.1)

### CODE REVIEW

**Comment by aliceh** [comment]
Tests passed, but Jenkins failed to archive artifacts in s.group test. Good to merge.

**Comment by dkavanagh** [comment]
looks good to me

File: eucaconsole/views/stacks.py [hunk 1]
```diff
@@ -793,6 +795,11 @@ def get_vmtype_options(self):
         vmtypes = ChoicesManager(conn).instance_types(self.cloud_type)
         return vmtypes
 
+    def get_availability_zone_options(self):
+        conn = self.get_connection()
+        zones = ChoicesManager(conn).availability_zones(self.cloud_type, add_blank=False)
```
Comment (dkavanagh) [inline eucaconsole/views/stacks.py]: I'm pondering the add_blank here.  The rest default to True. We've chosen to not provide a default selection here unless the template has a default or there is only 1 value available.

Comment (dkavanagh) [inline eucaconsole/views/stacks.py]: Now, I'm noticing how other methods to gather values were just using the connection directly and don't have a blank. Yes, setting this to False is correct.

**Pull request merged**